src/calendarUtils.ts

```diff
@@ -188,13 +188,15 @@ const getWeekDay: Function = ({date}: {date: Date}): WeekDay => {
   };
 };
 
-export const getWeekViewHeader: Function = ({viewDate, weekStartsOn}: {viewDate: Date, weekStartsOn: number}): WeekDay[] => {
-
+export const getWeekViewHeader: Function = ({viewDate, weekStartsOn, excluded}:
+    {viewDate: Date, weekStartsOn: number, excluded: number[]}): WeekDay[] => {
   const start: Date = startOfWeek(viewDate, {weekStartsOn});
   const days: WeekDay[] = [];
   for (let i: number = 0; i < DAYS_IN_WEEK; i++) {
-    const date: Date = addDays(start, i);
-    days.push(getWeekDay({date}));
+    if (!excluded || !excluded.some(e => i === e)) {
```

Comment:
You can just use a default parameter for excluded i.e. `excluded = []` in the function definition